### DISCUSSION TITLE
How can I unblock my IP in codesanbox.io?

### DISCUSSION THREAD

**Discussion by DigitalBrainJS** [codesandbox/codesandbox-client#6715] [General] [answered]
I'm not sure this is the right place, but I sent a request via the codesanbox.io feedback form almost a week ago, and still no answer/result.
About two weeks ago, I noticed that my IP got banned. So the question is how to contact the administration/support of the site to unblock it? 
![image](https://user-images.githubusercontent.com/12586868/170488496-a81e68fa-f489-498e-a85d-c928215146e6.png)

**Comment by CodeSandboxTeam** [comment]
Thank you for flagging this, we're investigating now



--- original message ---
On May 26, 2022, 1:37 PM GMT+1 ***@***.*** wrote:



I'm not sure this is the right place, but I sent a request via the codesanbox.io feedback form almost a week ago, and still no answer/result.

About two weeks ago, I noticed that my IP got banned. So the question is how to contact the administration/support of the site to unblock it?


—

Reply to this email directly, view it on GitHub, or unsubscribe.

You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
--- end of original message ---

**Comment by andrasbacsai** [comment, marked as answer]
Hey, can you please send me your IP address so we can unban you?

**Reply by andrasbacsai** [reply]
Removed from Cloudflare, could you please check now?

**Reply by DigitalBrainJS** [reply]
Yes, now it works! Thanks a lot!

**Comment by D-Daveloper** [comment]
mine is also banned how do i unban it please help!!
